mitodl/mit-open
# Faster yarn commands (linting, formatting)

#### What are the relevant tickets?
None

#### What's this PR do?
This PR makes our yarn commands faster.

| cmd           | before | now  |
|---------------|--------|------|
| fmt-fix       | 39     | 15s  |
| lint-fix      | 25s    | 9s   |
| scss-lint-fix | 14s    | 3.5s |

#### How should this be manually tested?
1. Introduce a linting error (like an unused, unexported variable, `const five = 5`) and run `docker compose exec watch yarn lint-check`. You should get an error. (Check the exit code with `echo $?`.) 
2. Repeating for formatting and SCSS linting errors.
